src/components/transport_manager/src/transport_adapter/transport_adapter_impl.cc

```diff
@@ -751,21 +751,20 @@ void TransportAdapterImpl::ConnectFailed(const DeviceUID& device_handle,
 void TransportAdapterImpl::RemoveFinalizedConnection(
     const DeviceUID& device_handle, const ApplicationHandle& app_handle) {
   const DeviceUID device_uid = device_handle;
-  const ApplicationHandle app_uid = app_handle;
```

Comment:
@BTPankow 
This handle also using inside of `connections_.find` and this delete lead to build fail.